react/src/components/CommentaryList/strings.js

```diff
@@ -0,0 +1,5 @@
+const strings = {
+
+}
```

Comment:
I'd create these files whenever they are needed, so we simplify the folder structure :)